02week/pigLatin.js

```diff
@@ -7,17 +7,77 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+// write a function that takes in 1 string (word) global
+pigLatin = (word) => {
+  //define vowels
+  const vowels = ["a", "e", "i", "o", "u"];
+  //once vowels are identified, split the word and create a new variable
+  let splitWord = word.split('');
 
```

Comment:
You currently have two functions name "pigLatin", which will throw errors. You do not need the first one. Everything inside that function can be put inside your second function so that you only have one function name "pigLatin"